test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -2008,7 +2019,7 @@ describe('isConnectedState()', () => {
   });
 
   it('should return "false" for non-connected states', () => {
-    expect(isConnectedState(undefined)).toBe(false);
+    expect(isConnectedState(undefined as any)).toBe(false);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    expect(isConnectedState(undefined as unknown)).toBe(false);
```